<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>394</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>535</width>
    <height>394</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheapOVR configuration utility</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color:white;
background-color: #242424;
}
QComboBox{
border: none;
border-radius: 6px;
background-color: #4A4A4A;

}
QComboBox:hover{
background-color: #545454;
}
QComboBox:pressed{
background-color: #727272;
}
QComboBox::drop-down{
border: none;
}
QComboBox::down-arrow{
image: url(:/images/pan-down-symbolic.svg);
margin-right: 15px;
}
QComboBox QListView{
border: none;
padding: 6px;
background-color: #383838;
border-radius: 12px;
}

QComboBox QListView:item{
height: 34px;
border-radius: 6px;
background: transparent;

}
QComboBox QListView::item:hover{
    background: #383838;
    color: white;
}
QComboBox QListView::item:selected{
    background: #383838;
    color: white;
}
QToolButton {
background-color: #4A4A4A;
border:none;
border-radius:5px;
}
QToolButton:hover{
background-color: #545454;
}

QToolButton:pressed{
background-color: #727272;
}
QPushButton {
background-color: #4A4A4A;
border:none;
border-radius:5px;
}
QPushButton:hover{
background-color: #545454;
}
QPushButton:pressed{
background-color: #727272;
}
QLineEdit {
background-color: #4A4A4A;
border:none;
border-radius:5px;
padding-left: 6px;
padding-right: 6px;
}
QLineEdit:read-only{
	color: #c9c9c9;
	background-color: #2b2b2b;
}
QLineEdit::hover {
	background-color: #545454;
}
QCheckBox::indicator {
       height: 26px;
       width: 46px;
   }

   QCheckBox::indicator:checked {
       image: url(&quot;:/images/toggle-on.svg&quot;);
   }
  QCheckBox::indicator:unchecked {
       image: url(&quot;:/images/toggle-off.svg&quot;);
   }
QRadioButton::indicator { width: 22px; height: 22px;}

QRadioButton::indicator::checked { image: url(&quot;:/images/radiobutton-checked.svg&quot;); }
QRadioButton::indicator::unchecked { image: url(&quot;:/images/radiobutton-unchecked.svg&quot;); }

QTabWidget::pane {
border: none;

}
QTabWidget::tab-bar{
	left: 5px;
}
QTabBar::tab {
	min-height: 34px;
	border-radius: 5px;
    padding-right: 6px;
	padding-left: 6px;
	margin: 3px;
}
QTabBar::tab:selected{
	background-color: #2F2F2F;
}
QTabBar::tab:hover{
	background-color: #4B4B4B;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>535</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Driver configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="seriallabel">
            <property name="text">
             <string>Serial port:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QComboBox" name="serialCombo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>34</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="reloadBtn">
              <property name="minimumSize">
               <size>
                <width>34</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string>Reload</string>
              </property>
              <property name="icon">
               <iconset resource="app_assets.qrc">
                <normaloff>:/images/reload.svg</normaloff>:/images/reload.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Work mode:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="uartRadio">
              <property name="text">
               <string>UART (Serial)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="bleRadio">
              <property name="text">
               <string>Bluetooth Low Energy (Beta)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="udpRadio">
              <property name="text">
               <string>Wi-Fi UDP (Beta)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="wsRadio">
              <property name="text">
               <string>Wi-Fi WebSocket</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Wi-Fi mode settings</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="portLine">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>34</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>IP:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ipLine">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>34</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="savedrvBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Save driver config</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rotBtn">
          <property name="text">
           <string>Rotate the cube around the camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trackerBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Run tracker test</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Firmware configuration</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>219</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Wi-Fi credentials</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>SSID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>34</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>34</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="text">
            <string>Save Wi-Fi credentials</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="app_assets.qrc"/>
 </resources>
 <connections/>
</ui>
